jwql/website/apps/jwql/forms.py

```diff
@@ -58,20 +58,10 @@ def view_function(request):
 
 
 from jwql.database import database_interface as di
-from jwql.utils.constants import ANOMALY_CHOICES_PER_INSTRUMENT
-from jwql.utils.constants import ANOMALIES_PER_INSTRUMENT
-from jwql.utils.constants import APERTURES_PER_INSTRUMENT
-from jwql.utils.constants import DETECTOR_PER_INSTRUMENT
-from jwql.utils.constants import EXP_TYPE_PER_INSTRUMENT
-from jwql.utils.constants import FILTERS_PER_INSTRUMENT
-from jwql.utils.constants import GENERIC_SUFFIX_TYPES
-from jwql.utils.constants import GRATING_PER_INSTRUMENT
-from jwql.utils.constants import GUIDER_FILENAME_TYPE
-from jwql.utils.constants import JWST_INSTRUMENT_NAMES_MIXEDCASE
-from jwql.utils.constants import JWST_INSTRUMENT_NAMES_SHORTHAND
-from jwql.utils.constants import READPATT_PER_INSTRUMENT
-from jwql.utils.utils import get_config, get_rootnames_for_instrument_proposal, filename_parser
-from jwql.utils.utils import query_format
+from jwql.utils.constants import (ANOMALY_CHOICES_PER_INSTRUMENT, ANOMALIES_PER_INSTRUMENT, APERTURES_PER_INSTRUMENT, DETECTOR_PER_INSTRUMENT, EXP_TYPE_PER_INSTRUMENT,
```

Comment:
I'm always disappointed that PEP8 doesn't request an indentation for cases like this.